request_test.go

```diff
@@ -342,12 +410,47 @@ func TestSelectRequestDefaultValues(t *testing.T) {
 	assertBodyEqual(t, refBuf.Bytes(), req)
 }
 
+func TestSelectRequestSpaceByName(t *testing.T) {
+	var refBuf bytes.Buffer
+
+	resolver.nameUseSupported = true
+
+	refEnc := msgpack.NewEncoder(&refBuf)
+	err := RefImplSelectBody(refEnc, &resolver, "valid", defaultIndex, 0, 0xFFFFFFFF,
+		IterAll, []interface{}{}, nil, false)
+	if err != nil {
+		t.Errorf("An unexpected RefImplSelectBody() error %q", err.Error())
+		return
+	}
+
+	req := NewSelectRequest("valid")
+	assertBodyEqual(t, refBuf.Bytes(), req)
+}
+
+func TestSelectRequestIndexByName(t *testing.T) {
+	var refBuf bytes.Buffer
+
+	resolver.nameUseSupported = true
+
+	refEnc := msgpack.NewEncoder(&refBuf)
+	err := RefImplSelectBody(refEnc, &resolver, defaultSpace, "valid", 0, 0xFFFFFFFF,
+		IterAll, []interface{}{}, nil, false)
+	if err != nil {
+		t.Errorf("An unexpected RefImplSelectBody() error %q", err.Error())
+		return
```

Comment:
```suggestion
		t.Fatalf("An unexpected RefImplSelectBody() error %q", err.Error())
```

Reply:
Updated. Also updated every other `Errorf` + `return` use in this file (in a different commit).